internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -565,6 +604,9 @@ func resourceKustoClusterRead(d *pluginsdk.ResourceData, meta interface{}) error
 		d.Set("uri", props.URI)
 		d.Set("data_ingestion_uri", props.DataIngestionURI)
 		d.Set("engine", props.EngineType)
+		d.Set("allowed_ips", utils.FlattenStringSlice(props.AllowedIPRangeList))
+		d.Set("fqdns", utils.FlattenStringSlice(props.AllowedFqdnList))
+		d.Set("outbound_network_restriction_enabled", props.RestrictOutboundNetworkAccess == kusto.ClusterNetworkAccessFlagEnabled)
```

Comment:
i think in this case its best to keep this as the property name:
```suggestion
		d.Set("restrict_outbound_network_access", props.RestrictOutboundNetworkAccess == kusto.ClusterNetworkAccessFlagEnabled)
```
or
```suggestion
		d.Set("outbound_network_access_restricted", props.RestrictOutboundNetworkAccess == kusto.ClusterNetworkAccessFlagEnabled)
```

Reply:
@katbyte  thanks for the great suggestions, I've updated the PR, please take another look, thanks!